environment_setup/setup_networking.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+# At the time of creating this script, we have team members with enp3s0f1,
+# wlp3s0, wlp7s0, eth1, eth0 and a plethora of other network interfaces 
+#
+# This script will create two virtual interfaces tbots_wifi and tbots_eth
+# and then bridge those with the appropriate physical interfaces on the users
+# system.
+#
+# This script can be rerun to reconfigure the interfaces.
+echo "================================================================"
+echo " Networking Interface Setup"
+echo "================================================================"
+
+TBOTS_WIFI_INTERFACE=tbots_wifi
+TBOTS_ETHERNET_INTERFACE=tbots_eth
+
+echo "Removing tbots network interfaces, if they exist"
+sudo ip link set dev $TBOTS_WIFI_INTERFACE down
+sudo ip link set dev $TBOTS_ETHERNET_INTERFACE down
+sudo brctl delbr $TBOTS_WIFI_INTERFACE
+sudo brctl delbr $TBOTS_ETHERNET_INTERFACE
```

Comment:
Can we check for errors on all of these commands and the ones below? (This would catch things like `sudo: brctl: command not found`)